src/com/lftechnology/java/training/srijan/basics/football/PointsStructure.java

```diff
@@ -0,0 +1,146 @@
+package com.lftechnology.java.training.srijan.basics.football;
+
+import java.util.logging.Logger;
+
+
+public class PointsStructure {
+	private static final int WIN;
+	private static final int LOSE;
+	private static final int DRAW;
+	private int points;
+	private int counts;
+	private int wins;
+	private int draws;
+	private int lose;
+
+	static {
+		WIN = 3;
+		LOSE = 0;
+		DRAW = 1;
+	}
+
+	public PointsStructure(String club) {
+		this.points = 0;
+		this.counts = 1;
+		this.wins = 0;
+		this.draws = 0;
+		this.lose = 0;
+	}
+
+	private static final Logger LOGGER = Logger.getLogger(PointsStructure.class.getName());
+
+	/**
+	 * <p>
+	 * This method helps to get Points from an Array
+	 * </p>
+	 * 
+	 * @param numberOfMatches
+	 *            {@link Integer}
+	 * @param fixturesWithScore
+	 *            {@link String}
+	 * @author Srijan Bajracharya <srijanbajracharya@lftechnology.com>
+	 */
+	public void getPoints(int numberOfMatches, String[] fixturesWithScore) {
+		String result = "";
+		int stringHomePosition = 0;
+		int stringAwayPosition = 2;
+		for (int i = 0; i < numberOfMatches; i++) {
+			if (fixturesWithScore[stringHomePosition].equals("Manchester United")) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move the "Manchester United" string literal on the left side of this string comparison. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1132)